tokio/src/lib.rs

```diff
@@ -381,10 +381,6 @@ cfg_signal_internal! {
     pub(crate) mod signal;
 }
 
-cfg_stream! {
-    pub mod stream;
```

Comment:
Question: Should we keep an empty module w/ documentation explaining the stream functionality will return once `Stream` stabilizes in 1.0 and `tokio-stream` is the current work around? This way, searching for `stream` in the API docs will find *something*.

Reply:
That sounds like a good idea.

Reply:
We can follow up.